mods/cnc/maps/gdi06/rules.yaml

```diff
@@ -73,16 +73,16 @@ SAM:
 RMBO:
 	MustBeDestroyed:
 		RequiredForShortGame: true
-	AutoTarget:
-		TargetWhenIdle: false
-		TargetWhenDamaged: true
 	Health:
 		HP: 150
 
 RMBO.easy:
 	Inherits: RMBO
 	Health:
 		HP: 300
+	AutoTarget:
+		EnableStances: false
```

Comment:
These appear to have jumped from RMBO to RMBO.easy.  I don't think this is intentional?